FirebaseFunctions/Sources/Callable+Codable.swift

```diff
@@ -159,10 +167,12 @@ public struct Callable<Request: Encodable, Response: Decodable> {
   public func callAsFunction(_ data: Request) async throws -> Response {
     return try await call(data)
   }
+}
 
+public extension Callable {
   // TODO: Look into handling parameter-less functions.
   @available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *)
-  public func stream(_ data: Request) -> AsyncThrowingStream<Response, Error> {
+  func stream(_ data: Request?=nil) -> AsyncThrowingStream<Response, Error> {
```

Comment:
keep public and add spaces
```suggestion
  public func stream(_ data: Request? = nil) -> AsyncThrowingStream<Response, Error> {
```